lib/core/versions/latest/util/Jws.ts

```diff
@@ -22,14 +78,55 @@ export default class Jws {
     return verified;
   }
 
+  /**
+   * Signs the given protected header and payload as a JWS.
+   *
+   * @param payload If the given payload is of string type, it is assumed to be encoded string;
+   *                else the object will be stringified and encoded.
+   */
+  public static async sign (
+    protectedHeader: any,
+    payload: any | string,
```

Comment:
Any covers string, unless you think it's better to be explicit.

Reply:
Removed.